ixmp/core.py

```diff
@@ -250,6 +250,53 @@ def add_unit(self, unit, comment='None'):
         """
         self._jobj.addUnitToDB(unit, comment)
 
+    def regions(self):
+        """Return all regions defined for the IAMC-style timeseries format
+        including known synonyms.
+
+        Returns
+        -------
+        pd.Dataframe
```

Comment:
```
:class:`pandas.DataFrame`
```
as in, e.g. the docstring for `TimeSeries.timeseries()`.

Reply:
done